### ISSUE TITLE
Lost connection

### ISSUE THREAD

**Issue by thiagogsr** [devspace/devspace-user-requests#94]
Hi Zeno

I don't know what is happening but sometimes my devspace stops the auto refresh, maybe after sleep my macbook or toggle the internet connection. It works after refresh the page.

UPDATE:
I'm using Safari, I don't know if it happens in others browsers.

Thanks

**Comment by lucasbento** [comment]
An alert claiming "Lost connection" would be good for these situations.

Same thing happens here when I close my notebook for some time.

**Comment by ftonato** [comment]
Hi **:)**

My sugestion to solve this problem is to use this feature: [Online State](https://whatwebcando.today/online-state.html)

**Comment by zenorocha** [comment]
We already use this API. Try turning off your connection and turning on again ;)

![on-off](https://cloud.githubusercontent.com/assets/398893/16048755/21b37a5a-320a-11e6-9e0a-1e24f2c2b4ca.gif)

**Comment by thiagogsr** [comment]
It is not the problem. There is connection but the feed is not
automatically updated.

**Comment by zenorocha** [comment]
Are you using the desktop version?

**Comment by thiagogsr** [comment]
No @zenorocha, the web version in Safari. Example: when I close my macbook and open it again, the feed is not updated until I reload the page.

**Comment by zenorocha** [comment]
Ok, let me check @thiagogsr.

@lucasbento @ftonato are you using Safari as well?

**Comment by ftonato** [comment]
@zenorocha I don't =(

**Comment by zenorocha** [comment]
Are you having the same problem in other browsers?

**Comment by thiagogsr** [comment]
I just use Safari but I'm starting it in another browser.

**Comment by ftonato** [comment]
In Google Chrome it's working normally.

**Comment by thiagogsr** [comment]
It not happens in Google Chrome

**Comment by lucasbento** [comment]
@zenorocha I'm using Google Chrome but I guess my issue may be a little different, it does happen here.

If I don't use the notebook for a while and it stays blocked, DevSpace won't update untill I refresh it as @thiagogsr mentioned.

I'll take a look at the console if that happens again.

**Comment by thiagogsr** [comment]
Yes, It has happening in Google Chrome too. I don't know if it helps, but see the console screenshot below.

![image](https://cloud.githubusercontent.com/assets/1802025/16138105/694e2db2-3410-11e6-8732-54027bd353b3.png)
